CHANGELOG.md

```diff
@@ -1,5 +1,30 @@
 # CHANGELOG
 
+## v1.6.0
+
+#### API Changes
+
+*Shopper Login*
+
+* New Endpoints
+
+  | **Endpoint Name** | **Description** |
+  | ------------- |-------------|
+  | getPasswordResetToken | Request a reset password token |
+  | resetPassword | Creates a new password |
```

Comment:
Is `resetPassord` moved from shopper customers to shopper login raml?

Reply:
I don't believe so? Otherwise it would've triggered on the diff and we would've seen a breaking change for operation removed for `resetPassword` under shopper customers. I took a look at the RAML files for both shopper customers and shopper login and it seems that they both have `resetPassword`